src/test/java/org/openmrs/contrib/qaframework/page/AllergyPage.java

```diff
@@ -0,0 +1,29 @@
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.uitestframework.page.Page;
```

Comment:
to my plan, addOrEdit should be the same page as this, the other second is the allergies which rather should include allergy list etc